### PR TITLE
Custom DNS support

### LINKED ISSUE #253: Add a "switch DNS" menu entry (after setup)
Hello,

Can you add option in menu like " switch dns " for switching to another DNS custom or build in menu from start up?

Best Regards
Mateusz

### PR DESCRIPTION
**Pull request by sayem314**
Followup https://github.com/Angristan/OpenVPN-install/pull/208

Closes https://github.com/Angristan/OpenVPN-install/issues/253 and ~https://github.com/Angristan/OpenVPN-install/issues/258~

### CODE REVIEW

File: openvpn-install.sh [hunk 1]
```diff
@@ -572,6 +588,12 @@ ifconfig-pool-persist ipp.txt" >> /etc/openvpn/server.conf
 		echo 'push "dhcp-option DNS 176.103.130.130"' >> /etc/openvpn/server.conf
 		echo 'push "dhcp-option DNS 176.103.130.131"' >> /etc/openvpn/server.conf
 		;;
+		10) # Custom DNS
+		echo """push "dhcp-option DNS $DNS1"""" >> /etc/openvpn/server.conf
```
Comment (cezar97) [inline openvpn-install.sh]: Why all that quotes, as there aren't any special characters that need escaping. Can't just be `'push "dhcp-option DNS $DNS1"'`?

Comment (sayem314) [reply openvpn-install.sh]: The inside quotes are mandatory to paste them on file. With single quotes I cannot pass variables.

**Comment by angristan** [comment]
This is not related to #253 btw.

**Comment by angristan** [comment]
`^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$` is not a regex for IPv4

**Comment by cezar97** [comment]
@Angristan no it's only for numbers. But it's enough as it's the user that has to type it correctly.

**Comment by angristan** [comment]
I had to make another PR because of merge conflicts: https://github.com/angristan/openvpn-install/pull/470

**Pull request closed without merging**